packages/date-time/src/components/WeeklyDayPicker/examples/WeeklyDayPicker.Inline.Expandable.Example.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { WeeklyDayPicker, DayOfWeek, addDays, IWeeklyDayPickerProps } from '@uifabric/date-time';
+
+import * as styles from './WeeklyDayPicker.Example.scss';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+
+const DayPickerStrings = {
+  months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+  shortDays: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+  goToToday: 'Go to today',
+  weekNumberFormatString: 'Week number {0}',
+  prevMonthAriaLabel: 'Previous month',
+  nextMonthAriaLabel: 'Next month',
+  prevYearAriaLabel: 'Previous year',
+  nextYearAriaLabel: 'Next year',
+  prevYearRangeAriaLabel: 'Previous year range',
+  nextYearRangeAriaLabel: 'Next year range',
+  closeButtonAriaLabel: 'Close'
+};
+
+export interface IWeeklyDayPickerInlineExpandableExampleState {
+  selectedDate?: Date;
+  expanded?: boolean;
+}
+
+export interface IWeeklyDayPickerInlineExpandableExampleProps extends Omit<IWeeklyDayPickerProps, 'strings'> {
+  showExpandButton?: boolean;
+  showNavigateButtons?: boolean;
+}
+
+export class WeeklyDayPickerInlineExpandableExample extends React.Component<
+  IWeeklyDayPickerInlineExpandableExampleProps,
+  IWeeklyDayPickerInlineExpandableExampleState
+> {
+  public constructor(props: IWeeklyDayPickerInlineExpandableExampleProps) {
+    super(props);
+
+    this.state = {
+      selectedDate: new Date(),
+      expanded: false
+    };
+
+    this._onSelectDate = this._onSelectDate.bind(this);
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div className={styles.wrapper}>
+        {
+          <div>
+            Selected date(s): <span>{!this.state.selectedDate ? 'Not set' : this.state.selectedDate.toLocaleString()}</span>
+          </div>
+        }
+        {(this.props.minDate || this.props.maxDate) && (
+          <div>
+            Date boundary:
+            <span>
+              {' '}
+              {this.props.minDate ? this.props.minDate.toLocaleDateString() : 'Not set'}-
+              {this.props.maxDate ? this.props.maxDate.toLocaleDateString() : 'Not set'}
+            </span>
+          </div>
+        )}
+        {this.props.restrictedDates && (
+          <div>
+            Disabled date(s):
+            <span>
+              {' '}
+              {this.props.restrictedDates.length > 0
+                ? this.props.restrictedDates.map((d: Date) => d.toLocaleDateString()).join(', ')
+                : 'Not set'}
+            </span>
+          </div>
+        )}
+        {this.props.showExpandButton && (
+          <div>
+            <DefaultButton className={styles.button} onClick={this._expand} text="Expand/collapse" />
```

Comment:
This button needs aria-label. And something else to convey the state of the picker - whether it's expanded/collapsed. I ran Narrator over this and nothing's announced when this action is performed.

Reply:
Why would we need an aria-label on a button that has text in it? Shouldn't narrator read out the text if there isn't an aria-label specified? It's true though that narrator doesn't announce anything when performing the action, I'm not really sure what the right aria behavior is there.

https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions says that we can put the aria-expanded attribute on the button, and doing so seems to announce expand/collapse correctly.